test/hello_world/Dockerfile

```diff
@@ -1,31 +1,34 @@
-FROM ros:indigo-ros-base
+FROM ros:indigo
+
+ENV ROS_WSPACE "/ros_ws"
+WORKDIR "${ROS_WSPACE}"
 RUN apt-get update \
  && apt-get install -y build-essential qtbase5-dev \
-      ros-indigo-ros-tutorials \
-      ros-indigo-common-tutorials \
       vim \
       wget \
+      curl \
+      python-catkin-tools \
+      python-rosinstall-generator \
+      python-pip \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists
 
-RUN mkdir /root/overlay_ws \
-    && cd /root/overlay_ws \
+RUN . /opt/ros/${ROS_DISTRO}/setup.sh \
+    && cd ${ROS_WSPACE} \
     && wstool init src \
-    && cd /root/overlay_ws/src \
+    && cd ${ROS_WSPACE}/src \
     && wstool set ros_tutorials --git git://github.com/ros/ros_tutorials.git -y \
-    && wstool update 
-
-RUN cd /root/overlay_ws \
+    && wstool update \
+    && cd ${ROS_WSPACE} \
     && /bin/bash -c "source /opt/ros/indigo/setup.bash && catkin_make"
 
 # create entrypoint
-ENV ROS_WSPACE "/root/overlay_ws"
 WORKDIR "${ROS_WSPACE}"
 RUN echo "#!/bin/bash \n\
 set -e \n\
-source \"/opt/ros/indigo/setup.bash\" \n\
+source \"/opt/ros/${ROS_DISTRO}/setup.bash\" \n\
 source \"${ROS_WSPACE}/devel/setup.bash\" \n\
 exec \"\$@\"" > "${ROS_WSPACE}/entrypoint.sh" \
  && chmod +x "${ROS_WSPACE}/entrypoint.sh"
-ENTRYPOINT ["/root/overlay_ws/entrypoint.sh"]
+ENTRYPOINT ["/ros_ws/entrypoint.sh"]
```

Comment:
Should this be `${ROS_SPACE}`?

Reply:
If I set it to `ENTRYPOINT ["${ROS_WSPACE}/entrypoint.sh"]` it gives me error that it doesn't recognize `${ROS_WSPACE}`

Reply:
Interesting. Okay. `/ros_ws/` is fine for now.